### DISCUSSION TITLE
Перевод с английского на русский

### DISCUSSION THREAD

**Discussion by bogdal1993** [wladradchenko/wunjo.wladradchenko.ru#10] [Ideas]
Подскажите пожалуйста, есть сейчас возможность с помощью вашего проекта делать перевод английского голоса на русский с сохранением голоса и интонации?

**Comment by wladradchenko** [comment]
> Подскажите пожалуйста, есть сейчас возможность с помощью вашего проекта делать перевод английского голоса на русский с сохранением голоса и интонации?

Привет. На данный момент нет модели для клонирования голоса на русский язык, её надо ещё обучить. Есть модели для клонирования с любого языка на английский и китайский. Дополнительно, общая интонация сохраняется, чувствуется, что голос похож (регулировка скорости говорения будет в 1.6), но в голосе не передаются эмоции, например смех, поэтому при клонировании, частично интонация будет потеряна для любого языка + плюс там стоит нормализации, чтобы высота звучание была одинаковой для всех букв. В обновлении 1.6, я буду смотреть, возможно ли накладывать интонацию на аудио дорожку с исходника

**Reply by BrasD99** [reply]
@wladradchenko, привет. Можно ли узнать ориентировочные сроки выкатки 1.6? Мне хотя бы примерную оценку. :)

**Reply by wladradchenko** [reply]
Ближайшее время планируется исправлять баги, которые возникают у пользователей в версии 1.5. Съемка видео, как что работает и как исправить популярные ошибки, заполнение документации. Потом отдых на месяц от проекта. Думаю под конец декабря выйдет.

**Comment by wladradchenko** [comment]
Сохранение голоса и интонации добавлено. Дальнейшее улучшение вероятнее связано с тем, что необходимо добавлять новые голоса в обучении модели синтезатора.

Релиз Wunjo AI 1.6.1.
